MIGRATION.md

```diff
@@ -910,247 +956,495 @@ export default {
 
 ### Angular: Support experimental zoneless support
 
-Angular 16 introduced support for zoneless mode. Storybook now supports this mode in experimental mode.
+Storybook now supports [Angular's experimental zoneless mode](https://angular.dev/guide/experimental/zoneless). This mode is intended to improve performance by removing Angular's zone.js dependency. To enable zoneless mode in your Angular Storybook, set the `experimentalZoneless` config in your `angular.json` file:
 
-To enable zoneless mode, set the `experimental_zoneless` flag to `true` in your `.storybook/main.ts` file:
+```diff
+{
+  "projects": {
+    "your-project": {
+      "architect": {
+        "storybook": {
+          ...
+          "options": {
+            ...
++           "experimentalZoneless": true
+          }
+        }
+        "build-storybook": {
+          ...
+          "options": {
+            ...
++           "experimentalZoneless": true
+          }
+        }
+      }
+    }
+  }
+}
+```
 
-````
-<h1>Migration</h1>
+### Addon-a11y: Replaced experimental `ally-test` tag behavior with `parameters.a11y.test`
 
-- [From version 8.x to 9.0.0](#from-version-8x-to-900)
```

Comment:
I caught this regression in this PR of mine:
https://github.com/storybookjs/storybook/pull/30909

So this is bringing back the correct content @valentinpalkovic